test/config.test.js

```diff
@@ -538,14 +537,6 @@ test('should overwrite option property active by ELASTIC_APM_ACTIVE', function (
   t.end()
 })
 
-test('should default serviceName to package name', function (t) {
-  var agent = new Agent()
-  agent.start()
-  t.strictEqual(agent._conf.serviceName, 'elastic-apm-node')
-  agent.destroy()
-  t.end()
-})
```

Comment:
REVIEW NOTE: This test is a duplicate of "serviceName/serviceVersion zero-conf: valid" below.